monitor-stddev.py

```
@@ -644,31 +813,50 @@ def monitor_website(
+        try:
+            saver = SftpSaver(
+                host=sftp_config["host"],
+                port=sftp_config["port"],
+                username=sftp_config["username"],
+                key_path=sftp_config["key_path"],
+                remote_dir=sftp_config["remote_dir"],
+                known_hosts_path=sftp_config["known_hosts_path"],
+                save_name_json=save_name_json,
+                save_name_html=save_name_html,
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)